index.html

```diff
@@ -83,26 +112,64 @@
                     <!-- Stop ignoring -->
                     {% for student in site.data.students %}
                     <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
-                        <div class="card stu-card">
-                            <div class="avatar">
-                                {% if student.image %}
-                                <img src="images/students/{{student.image}}"
-                                alt="{{student.name}}"
-                                width="200px" height="200px">
-                                {% else %}
-                                <img src="images/mentors/member.png"
-                                alt="{{student.name}}"
-                                width="200px" height="200px">
-                                {% endif %}
-                                <p style="padding-top:3px;width: 200px;display: inline-block;margin-top: 9px;">
-                                    <i class="fa fa-code"></i>
-                                    &nbsp;
-                                    <span class="stu-name card-text" title="{{student.name}}">
-                                        {{student.name}}
-                                    </span>
-                                </p>
+                        <div class="card-wrapper">
+                            <div class="flip-container" ontouchstart="this.classList.toggle('hover');">
+                                <div class="flipper">
+                                    <div class="front">
+                                        <!-- front content -->
+                                        <div class="card stu-card">
```

Comment:
@VamsiSangam I was telling to do this because currently the card flips even when you are not on the actual card, 
placing the flip code inside the avatar class will ensure that .
Cheers.
![screenshot from 2017-01-18 12-01-31](https://cloud.githubusercontent.com/assets/13910561/22099185/43f8c15e-de50-11e6-99f2-dc9d4a7b7335.png)
![screenshot from 2017-01-18 12-01-42](https://cloud.githubusercontent.com/assets/13910561/22099186/4478dbbe-de50-11e6-9a69-c93f6dec997c.png)